index.html

```diff
@@ -0,0 +1,136 @@
+
+<!DOCTYPE html>
+<html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <!-- link to CSS -->
+    <link rel="stylesheet" href="stylesheet.css">
+  <!--link to fontawesome icons  -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://kit.fontawesome.com/9f86628c0e.js" crossorigin="anonymous"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Title of document -->
+    <title>Webpage</title>
+    <header>
+      <!-- Navbar -->
+        <div id="topnav">
+          <!-- Home button and icon -->
+          <a href="#home"><i class="fa fa-fw fa-home" alt ="House symbol"></i> Home</a>
+           <!-- About me button -->
+            <a href="#about-me">About</a>
+            <!-- SOC button and icon -->
+            <a href="#SOC"><i class="fas fa-desktop"></i> SOC</a>
+            <!-- Contact button and icon -->
+            <a href="#contact">Contact</a>
+          </div>
+      </header>
+</head>
+<body>
+  <section class="home" id= "home">
```

Comment:
It only looks like the class of `home` is being used. If there's ununsed code in a site, it's clearer to delete it rather than have it clutter the document. For the difference between classes and IDs, think about whether it is being applied to one element or to many different elements.

```suggestion
  <section class="home">
```